tensorrt_llm/_torch/pyexecutor/sampler/sampler_features.py

```
@@ -0,0 +1,745 @@
+    # Indices of unique bias tensors
+    #
+    # NB: hash(torch.Tensor) is equivalent to id(torch.Tensor), and does not
+    #     depend on tensor contents, cf. https://github.com/pytorch/pytorch/issues/2569
+    bias_to_index: dict[torch.Tensor, int] = defaultdict(provision_bias_index)
+
+    # Source indices for bias application
+    bias_gather_indices: list[int] = []
+
+    # Collect bias information
+    req_bias = None
+    for i, (req, steps) in enumerate(zip(requests, request_steps_list)):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)